irohad/ametsuchi/impl/rocksdb_common.hpp

```diff
@@ -996,6 +1000,8 @@ namespace iroha::ametsuchi {
   template <typename RetT, kDbOperation kOp, kDbEntry kSc, typename... Args>
   inline expected::Result<std::optional<RetT>, DbError> dbCall(
       RocksDbCommon &common, Args &&... args) {
+    static_assert(kOp != kDbOperation::kDel or kSc == kDbEntry::kCanExist,
```

Comment:
remove plz.

Reply:
at the moment I wrote this it was not in execOperation()